cohort-engine-api-web/src/main/java/com/ibm/cohort/engine/api/service/CohortEngineRestHandler.java

```diff
@@ -236,6 +238,7 @@
 			"<p>The <code>patientIds</code> is a comma separated list of patients to run. Supplying a single patient does not need any trailing commas.</p>" +
 			"<p>The parameter types and formats are described in detail in the <a href=\"http://alvearie.io/quality-measure-and-cohort-service/#/user-guide/parameter-formats?id=parameter-formats\">user guide</a>.</p>" +
 			"<p>The <code>loggingLevel</code> will determine how much and what type of logging to provide. The options are NA, COVERAGE, and TRACE.</p>" +
+			"<p>For more detailed information about the cohort endpoint, check out our <a href=\"http://alvearie.io/quality-measure-and-cohort-service/#/user-guide/rest_endpoints?id=cohort-endpoint\">documentation</a> here.</p>" +
```

Comment:
I see you have links point to sections internally in user guide for rest endpoints in each of the EXAMPLE REQUEST_DATA JSON Strings.  If you removed anything REST specific from user guide those should probably not be there either??